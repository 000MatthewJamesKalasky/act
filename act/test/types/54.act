deftype test1 (int<4> x) { }

deftype foo <: int<4> (bool x[4]) { }

deftype newtest <: test1()
+{ foo x; } 
{  }

newtest x;
