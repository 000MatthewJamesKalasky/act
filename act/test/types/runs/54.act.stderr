ERROR: File `54.act', line 5, col 8
	newtest: structure implementation cannot extend the port list!
